modules/compliance-supported-profiles.adoc

```diff
@@ -8,108 +8,127 @@
 The Compliance Operator provides the following compliance profiles:
 
 .Supported compliance profiles
-[cols="10%,40%,10%,40%,10%", options="header"]
+[cols="10%,40%,10%,10%,40%,10%", options="header"]
 
 |===
 |Profile
 |Profile title
+|Application
 |Compliance Operator version
 |Industry compliance benchmark
 |Supported architectures
 
 |ocp4-cis
 |CIS Red Hat OpenShift Container Platform 4 Benchmark
+|Platform
 |0.1.39+
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] footnote:cisbenchmark[To locate the CIS RedHat OpenShift Container Platform v4 Benchmark, go to  link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks] and type `Kubernetes` in the search box. Click on *Kubernetes* and then *Download Latest CIS Benchmark*, where you can then register to download the benchmark.]
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[1]^
 |`x86_64`
  `ppc64le`
  `s390x`
 
 |ocp4-cis-node
 |CIS Red Hat OpenShift Container Platform 4 Benchmark
+|Node ^[2]^
 |0.1.39+
-|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] footnote:cisbenchmark[]
+|link:https://www.cisecurity.org/cis-benchmarks/[CIS Benchmarks &#8482;] ^[1]^
 |`x86_64`
  `ppc64le`
  `s390x`
 
 |ocp4-e8
 |Australian Cyber Security Centre (ACSC) Essential Eight
+|Platform
 |0.1.39+
 |link:https://www.cyber.gov.au/acsc/view-all-content/publications/hardening-linux-workstations-and-servers[ACSC Hardening Linux Workstations and Servers]
 |`x86_64`
 
 |ocp4-moderate
 |NIST 800-53 Moderate-Impact Baseline for Red Hat OpenShift - Platform level
+|Platform
 |0.1.39+
 |link:https://nvd.nist.gov/800-53/Rev4/impact/moderate[NIST SP-800-53 Release Search]
 |`x86_64`
 
 |rhcos4-e8
 |Australian Cyber Security Centre (ACSC) Essential Eight
+|Platform
```

Comment:
All rhcos4 related rules should be Node Type.
@jhrozek Correct me if I am wrong.

Reply:
Good catch - I believe that is correct as well, based on upstream documentation. I'll wait for Jakub to verify as well.

Reply:
@xiaojiey is right, all the RHCOS profiles are Node type.